catalyst-gateway/bin/src/service/common/objects/hash.rs

```diff
@@ -0,0 +1,17 @@
+use poem_openapi::{types::Example, Object};
+
+#[derive(Object)]
+#[oai(example = true)]
+/// Blake2b256 hash wrapper.
+pub(crate) struct Hash {
+    /// Blake2b256 hash encoded in hex.
+    pub hash: String,
```

Comment:
need to add a validation, something like this ` #[oai(validator(max_length = 64, min_length = 64, pattern = "[0-9a-f]{64}"))]`

Reply:
Cool :).

Thanks for the suggestion. I'll do that for sure